tests/Application/config/sylius/1.10/bundles.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
I understand the Sylius Plugin Skeleton does this, but the maintenance burden with this approach is just too high in my opinion, therefore I think we should just upgrade the application to 1.10 and that's it.

Reply:
But what would happen for older versions ? We won't support them anymore ?

I can understand for 1.7 (perhaps also 1.8) but 1.9 is still quite recent and not everyone updated yet.

WDYT ?